db/schema.rb

```diff
@@ -175,6 +175,20 @@
     t.index ["tenant_id"], name: "index_containers_on_tenant_id"
   end
 
+  create_table "disks", force: :cascade do |t|
+    t.bigint "tenant_id", null: false
+    t.bigint "vm_id", null: false
+    t.bigint "volume_id"
+    t.string "device"
+    t.string "size"
+    t.string "status"
+    t.jsonb "extra"
+    t.index ["tenant_id"], name: "index_disks_on_tenant_id"
+    t.index ["vm_id", "device"], name: "index_disks_on_vm_id_and_device", unique: true
+    t.index ["vm_id"], name: "index_disks_on_vm_id"
```

Comment:
@agrare hm, we should eliminate all indexes like this, the uniq column should already cover this.